lib/processLayerStyles.js

```diff
@@ -28,10 +29,29 @@ const processLayerStyles = async (layerStyles, filepath, outputFolder, modifiers
             style: []
         }
         addOptionalProperty("background", data.style, layerFillWithShadows(layerStyle, outputFolder, sketchFile) )
-        addOptionalProperty("border", data.style, layerBorder(layerStyle) )
         addOptionalProperty("blur", data.style, layerBlur(layerStyle) )
-        // Apply the corner radius modifier
-        addOptionalProperty("cornerRadius", data.style, cornerRadius(layerStyle, data.name, modifiers.radius) )
+
+        // take the style, if there is a matching corner radius apply it.
+        let currentModifier = {modifier: null, score: 0};
+        for (const radiusModifier of modifiers.radius) {
+          const modName = getModifierNameWithoutModifier(radiusModifier);
+          const score = matchScore(style.name, modName)
+          if (currentModifier.score < score) {
+            currentModifier.modifier = radiusModifier;
+            currentModifier.score = score
+          }
```

Comment:
^ Hah, I'd forgotten what life without filter/sort/first was like!

Reply:
FYI filter and sort exist in js, first can be added with a polyfill